docs/docs/dev_docs/contracts/syntax/historical_access/how_to_prove_history.md

```diff
@@ -0,0 +1,103 @@
+---
+title: How to prove existence of historical notes and nullifiers
+---
+
+The Aztec Protocol uses an append-only Merkle tree to store the headers of all previous blocks in the chain as its leaves. This is known as an archive tree. You can learn more about how it works in the [concepts section](../../../../concepts/advanced/data_structures/trees.md#archive-tree).
+
+# History library
+
+The history library allows you to prove any of the following at a given block height before the current height:
+s
+* [Note inclusion](#note-inclusion)
```

Comment:
These links don't link to anything

Reply:
You could remove the links to each specific section and link to the Reference page.